src/cephadm/cephadm

```diff
@@ -1984,20 +1984,26 @@ def get_last_local_ceph_image(ctx: CephadmContext, container_path: str) -> Optio
     """
     :return: The most recent local ceph image (already pulled)
     """
+    # '|' special character is used to separate the output fields into:
+    #  - Repository@digest
+    #  - Image Tag
+    #  - Image creation date
     out, _, _ = call_throws(ctx,
                             [container_path, 'images',
                              '--filter', 'label=ceph=True',
                              '--filter', 'dangling=false',
-                             '--format', '{{.Repository}}@{{.Digest}}'])
+                             '--format', '{{.Repository}}@{{.Digest}}|{{.Tag}}|{{.CreatedAt}}'])
     return _filter_last_local_ceph_image(out)
 
 
 def _filter_last_local_ceph_image(out):
     # type: (str) -> Optional[str]
     for image in out.splitlines():
-        if image and not image.endswith('@'):
-            logger.info('Using recent ceph image %s' % image)
-            return image
+        if image and not image.isspace():
+            (digest, tag, created_date) = image.lstrip().split('|')
+            if digest and not digest.endswith('@'):
+                logger.info(f"Using ceph image with tag '{tag}' created on {created_date}\n{digest}")
```

Comment:
maybe the container image id would be more useful than then the creation time?

Reply:
Updated the PR to add the image_id plus other important enhancements.